internals/secrethub/flags.go

```diff
@@ -1,19 +1,13 @@
 package secrethub
 
 import (
-	"github.com/alecthomas/kingpin"
 	"github.com/secrethub/secrethub-cli/internals/cli"
 )
 
-// FlagRegisterer allows others to register flags on it.
-type FlagRegisterer interface {
-	Flag(name, help string) *cli.Flag
+func registerTimestampFlag(r *cli.CommandClause, p *bool) {
```

Comment:
If possible, I'd stick with using the `FlagRegisterer` interface as the parameter type for these functions. The main advantage of doing so is ensuring that the `register...Flag` functions only register a flag and do nothing else with the command, and also using this type simplifies testing.

Note that the interface can be slightly modified to better fit the method signatures of cobra.

Reply:
We decided to postpone implementing this change as it does not bring a lot of value. This idea will be assessed again after the major design changes we agreed on are implemented.